### ISSUE TITLE
nodetool cfstats missing info with LeveledCompactionStrategy

### ISSUE THREAD

**Issue by tzach** [scylladb/scylla#1119]
Scylla 0.19

```
cqlsh> CREATE KEYSPACE test WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };
cqlsh> CREATE TABLE test.users (user_name varchar PRIMARY KEY, birth_year bigint) WITH compaction = { 'class' :  'LeveledCompactionStrategy'  };
cqlsh> INSERT INTO test.users (user_name, birth_year) values ('tzach', 1970);
cqlsh> INSERT INTO test.users (user_name, birth_year) values ('nadav', 1975);
cqlsh> INSERT INTO test.users (user_name, birth_year) values ('shlomi', 1974);
cqlsh> INSERT INTO test.users (user_name, birth_year) values ('dor', 1974);
cqlsh> INSERT INTO test.users (user_name, birth_year) values ('avi', 1970);
cqlsh> INSERT INTO test.users (user_name, birth_year) values ('vlad', 1984);

$ nodetool cfstats test users
nodetool: Unknown keyspace: users
See 'nodetool help' or 'nodetool help <command>'.
[centos@ip-172-31-38-1 ~]$ nodetool cfstats test 
Keyspace: test
	Read Count: 0
	Read Latency: NaN ms.
	Write Count: 6
	Write Latency: 0.014166666666666666 ms.
	Pending Flushes: 0
		Table: users
		SSTable count: 0
		Space used (live): 0
		Space used (total): 0
		Space used by snapshots (total): 0
		Off heap memory used (total): 1048576
		SSTable Compression Ratio: 0.0
		Number of keys (estimate): 0
		Memtable cell count: 6
		Memtable data size: 3498
		Memtable off heap memory used: 1048576
		Memtable switch count: 0
		Local read count: 0
		Local read latency: NaN ms
		Local write count: 6
		Local write latency: 0.010 ms
		Pending flushes: 0
		Bloom filter false positives: 0
		Bloom filter false ratio: 0.00000
		Bloom filter space used: 0
		Bloom filter off heap memory used: 0
		Index summary off heap memory used: 0
		Compression metadata off heap memory used: 0
		Compacted partition minimum bytes: 0
		Compacted partition maximum bytes: 0
		Compacted partition mean bytes: 0
		Average live cells per slice (last five minutes): 0.0
		Maximum live cells per slice (last five minutes): 0.0
		Average tombstones per slice (last five minutes): 0.0
		Maximum tombstones per slice (last five minutes): 0.0
```

Cassandra 2.1.8
```
Keyspace: test
	Read Count: 0
	Read Latency: NaN ms.
	Write Count: 1
	Write Latency: 0.212 ms.
	Pending Flushes: 0
		Table: users
		SSTable count: 1
		SSTables in each level: [1, 0, 0, 0, 0, 0, 0, 0, 0]
		Space used (live): 4762
		Space used (total): 4762
		Space used by snapshots (total): 0
		Off heap memory used (total): 33
		SSTable Compression Ratio: 0.8266666666666667
		Number of keys (estimate): 1
		Memtable cell count: 0
		Memtable data size: 0
		Memtable off heap memory used: 0
		Memtable switch count: 1
		Local read count: 0
		Local read latency: NaN ms
		Local write count: 1
		Local write latency: 0.213 ms
		Pending flushes: 0
		Bloom filter false positives: 0
		Bloom filter false ratio: 0.00000
		Bloom filter space used: 16
		Bloom filter off heap memory used: 8
		Index summary off heap memory used: 17
		Compression metadata off heap memory used: 8
		Compacted partition minimum bytes: 73
		Compacted partition maximum bytes: 86
		Compacted partition mean bytes: 86
		Average live cells per slice (last five minutes): 0.0
		Maximum live cells per slice (last five minutes): 0.0
		Average tombstones per slice (last five minutes): 0.0
		Maximum tombstones per slice (last five minutes): 0.0
```

Note the `SSTables in each level: [1, 0, 0, 0, 0, 0, 0, 0, 0]` info

**Comment by amnonh** [comment]
@slivne please reassign

**Comment by slivne** [comment]
I don;t know what the issue is - to know to who to reassign ...
why is this not working ?

**Comment by amnonh** [comment]
Column family sstable don't have an api for that sstable count per level
like cassandra has

**Comment by slivne** [comment]
Thanks

Can you provide a reference to the original code in cassandra

**Comment by amnonh** [comment]
ColumnFamilyStore.java: getSSTableCountPerLevel

**Comment by raphaelsc** [comment]
BTW, we can easily implement this by iterating through all sstables and incrementing a counter for its level. Like:

for (auto& sst : cf.get_sstables()) {

**Comment by tzach** [comment]
@raphaelsc @amnonh any update on this issue?

**Comment by dorlaor** [comment]
@raphaelsc first, please make sure it get's 1.3.1 asap 
Another thing Raphael, will DTCS have it too? Do you need to open a new bug

**Comment by raphaelsc** [comment]
This concept of level is generic (although only leveled strategy create sstables with level > 0), so the solution will work to all strategies. Alright... I will send a patch to it soon.

**Comment by slivne** [comment]
@raphaelsc any update on this issue ?

**Comment by raphaelsc** [comment]
@slivne Chasing a performance issue on clustering key filter, and I will move to this right after.

**Comment by penberg** [comment]
@slivne @dorlaor Is this scheduled for 1.3 on purpose? If so, we need to backport the fix.

**Comment by dorlaor** [comment]
@penberg well, I wanted it for 1.3.1, now it will be too late but let's get it for 1.3.2 since it's important and helpful

**Comment by dorlaor** [comment]
@penberg since we manage multiple versions, we may need to keep a bug closed on master while open on a branch (1.3.x)

**Comment by penberg** [comment]
@dorlaor Github closes the issue automatically once the commit is merged to master so keeping the issues open is manual work and therefore likely to be forgotten.

We usually backport fixes immediately and the only reason this one was delayed was because it doesn't really fit the usual "regression fixes only" pattern. However, I  see that "User Request" label there and the change is low risk, so I backported it now and the fix will appear in 1.3.2 and the upcoming 1.4.0.
